js/client-battle-tooltips.js

```diff
@@ -518,6 +518,12 @@ var BattleTooltips = (function () {
 			}
 			text += '</p>';
 		}
+		if (!this.battle.hardcoreMode && (pokemon.sleepTurns || pokemon.toxicTurns)) {
+			text += '<p class="section">';
+			if (pokemon.sleepTurns) text += 'Turns asleep: ' + pokemon.sleepTurns + '<br />';
+			if (pokemon.toxicTurns) text += 'Next Toxic damage: ' + pokemon.toxicTurns + '/16<br />';
```

Comment:
I would prefer if we didn't perpetuate the common misconception that Toxic damage is a flat X/16 when it's really floor(1/16) * X, where X caps at 15 turns.

Reply:
We should probably just compute the value, right?

Reply:
So `Math.floor(Math.min(pokemon.toxicTurns, 15) / 16 * 100)`?

Reply:
So `(Math.floor(pokemon.maxhp / 16) * Math.min(pokemon.toxicTurns, 15)) / pokemon.maxhp`?

Reply:
We don't have enemy Pokémon maxhp, so something subtler.

Reply:
Using 100 would work instead, for an estimate on both sides.

    (Math.floor(100 / 16) * Math.min(pokemon.toxicTurns, 15)) / 100

or for displaying it as a percentage instead of a decimal, leave out the `/ 100` bit.